docs/troubleshooting.mdx

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 9
+---
+
+# Troubleshooting
+
+## Supported Video Codecs
+
+Fishjam supports the following video codecs:
+
+- **H.264**
+- **VP8**
+
+## Default Codec
+
+Fishjam uses H.264 by default, however, for better compatibility, VP8 is set when you're using Room Manager on Sandbox.
```

Comment:
I would still consider adding here explanation that is not just about general compatibility, but to solve issue with android emulator. As I'm reading it now, I would be afraid to enable h.264 for production too.